packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -702,6 +688,8 @@ paths:
   });
 
   async function givenAServer(options?: {rest: RestServerConfig}) {
+    options = options || {rest: {port: 0}};
```

Comment:
Please note that `givenHttpServerConfig` sets the port to 0 by default. I think the following simpler version should work well too:

```ts
async function givenAServer(options: {rest?: RestServerConfig} = {}) {
   options.rest = givenHttpServerConfig(options.rest);
   // ...
}
```